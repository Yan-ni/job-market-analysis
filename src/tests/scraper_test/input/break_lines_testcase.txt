<div>
  <div>
    <p>paragraph test One<br>paragraph test Two</p>
    <br>
    <p>paragraph test Three<br></p>
    <p>paragraph test Four</p>
  </div>
</div>