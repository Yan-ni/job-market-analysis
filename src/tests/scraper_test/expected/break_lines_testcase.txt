paragraph test One
paragraph test Two
paragraph test Three

paragraph test Four